src/ScriptBuilder/Outbox/Create_MsSqlServer.sql

```diff
@@ -18,4 +18,37 @@ set @createTable = '
     )
 ';
 exec(@createTable);
+end
+
+if not exists
+(
+    select *
+    from sys.indexes
+    where
+        name = 'Index_DispatchedAt' and
+        object_id = object_id(@tableName)
```

Comment:
I know that this is not directly related to this PR but you can build `@tableName` using `QUOTENAME` instead of manually adding `[` and `]`